e2e/helper/helper.go

```diff
@@ -1,6 +1,7 @@
 package helper
```

Comment:
Renaming this package to utils maybe?
```suggestion
package utils
```

Reply:
then we need to rename all helpers to utils :) This is more about the convention I think. If everyone agrees to use utils instead of helpers then it would make sense to rename it as well. Otherwise, we would jump from utils to helpers and vice versa

Reply:
We can also use common for this)
I'll agree with both